src/Hosting/Hosting/src/Internal/WebHost.cs

```diff
@@ -342,21 +342,39 @@ public async Task StopAsync(CancellationToken cancellationToken = default)
         }
 
         public void Dispose()
```

Comment:
I know throwing in dispose is wrong but I'd rather do that then do sync over async...

Reply:
Talking about breaking changes, this one would be huge.

Reply:
It's not that big of a breaking change as it would you to manually dispose this feature which nobody does. What am I missing?

Reply:
This is not a feature, this is IWebHost implementation. The feature is below and yeah, it seems we can throw there.